va/va_fei.h

```diff
@@ -61,6 +61,102 @@ extern "C" {
 
 /**@}*/
 
+/** \brief Attribute value for VAConfigAttribStats */
+typedef union _VAConfigAttribValStats {
+    struct {
+        /** \brief Max number of past reference frames that are supported. */
+        uint32_t    max_num_past_references   : 4;
+        /** \brief Max number of future reference frames that are supported. */
+        uint32_t    max_num_future_references : 4;
+        /** \brief Number of output surfaces that are supported */
+        uint32_t    num_outputs               : 3;
```

Comment:
Can you explain what exactly is this "num_outputs"? I don't understand the comment "Number of output surfaces that are supported". Is it for "the number of output buffers(stat and mv)"?

Reply:
yes, it should be number of output buffers for stats and mv